benchmarks/perception/a3_stereo_image_proc/CMakeLists.txt

```diff
@@ -0,0 +1,48 @@
+cmake_minimum_required(VERSION 3.5)
```

Comment:
Is this file based on a project-wide standard?  Is there a way to standardize boilerplate files like this across different benchmarks?

Reply:
@vmayoral I saw that you were using this version in your CMake file in the a1 and a2 benchmarks so I kept it consistent in the a3 benchmark. Is there any particular reason you decided to use this version?

Reply:
Sorry, this was meant to be a comment regarding the entire file (and standard files like it), not the line in the file.  I was asking if there was a benchmark template folder that includes boilerplate files like this.  I believe this is a question for @vmayoral.